### ISSUE TITLE
[Question] Free Software

### ISSUE THREAD

**Issue by ghbjklhv1** [P2PVPS/p2pvps-client#16]
**Question**: Is P2PVPS fully and entirely [free/libre software](https://www.gnu.org/philosophy/free-sw.html)?

Will/are there any plans to change this?

**Comment by christroutner** [comment]
Yes.

**Comment by ghbjklhv1** [comment]
@christroutner To clarify, yes it is free/libre software?

And no there aren't any plans to change the license?

**Comment by christroutner** [comment]
Yes. It's free and libre software. Everything is licensed under the MIT
license. This will not change.
